include/ccmath/math/nearest/nearbyint.hpp

```diff
@@ -8,7 +8,64 @@
 
 #pragma once
 
+#include <ccmath/math/compare/isinf.hpp>
+#include <ccmath/math/compare/isnan.hpp>
+#include <ccmath/math/compare/signbit.hpp>
+#include <ccmath/math/nearest/trunc.hpp>
+#include <type_traits>
+
 namespace ccm
 {
+	/**
+	 * @brief The nearest integer value to num, according to the rounding mode FE_TONEAREST, is returned.
```

Comment:
Are we able to support all rounding modes? This should be pretty do-able with `fp_bits' and the `fp` folder in our support directory should have all the necessary tools to enable us to properly support all rounding modes. ^^

Reply:
I can surely look into it! I will look into the fp_bits in the project, thanks for bringing that up. I was initially unsure if this was something that was possible as the standard library implementation seemed very runtime reliant, this was why I opted to just implement what the default rounding mode seemed to be.